frontend/src/modules/modals/MultiStepWizard.tsx

```diff
@@ -40,36 +42,36 @@ import store from '../../store';
  * @prop {boolean} isSetvalEnabled Config to set if value is Set type
  * @prop {number | null} committedSetting Current value
  * @prop {number[]} alarmValues Alarm range values
- * @prop {number} setValue Set Value
+ * @prop {number} ValueDraft Set Value
  * @prop {number | null} minValue Minimum under which value cannot decrement
  * @prop {number | null} maxValue Maximum above which value cannot increment
  * @prop {number | null} alarmLimitMin Alarm limit Minimum Range value
  * @prop {number | null} alarmLimitMax Alarm limit Maximum Range value
  * @prop {number[]} alarmValuesActual Actual Alarm Value when Component was initalized
- * @prop {number} setValueActual Actual Set Value when Component was initalized
+ * @prop {number} ValueCurrent Actual Set Value when Component was initalized
  */
-interface Data {
+interface InternalState {
   stateKey: string;
   label: string;
   units: string;
   isAlarmEnabled: boolean;
   isSetvalEnabled: boolean;
   committedSetting?: number | null;
-  alarmValues: number[];
-  setValue: number;
+  alarmValues: Range;
```

Comment:
Should this actually be `alarmLimitsDraft`? Or does it serve a slightly different purpose?